zstackctl/zstackctl/ctl.py

```
@@ -1685,7 +1739,7 @@ def execute_on_peer(self, cmd, useSudo=False):
 
     def scp_to_peer(self, src_path, dst_path):
         shell("sudo -u %s scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no %s %s:%s" % (
-            self.ssh_exec_user, src_path, self.config['peerip'], "/tmp/dst_path"))
+            self.ssh_exec_user, src_path, format_url_host(self.config['peerip']), "/tmp/dst_path"))
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/dst_path"

(S108)